src/ngraph/runtime/intelgpu/intelgpu_backend.cpp

```diff
@@ -59,5 +64,43 @@ bool runtime::intelgpu::IntelGPUBackend::call(
     const vector<shared_ptr<runtime::TensorView>>& outputs,
     const vector<shared_ptr<runtime::TensorView>>& inputs)
 {
-    throw runtime_error("IntelGPUBackend::call: Not implemented yet");
+    validate_call(func, outputs, inputs);
+
+    FunctionInstance& instance = ocl_networks[func];
+    if (instance.ocl_network == nullptr)
+    {
+        if (!compile(func))
+        {
+            return false;
+        }
+    }
+
+    std::shared_ptr<cldnn::network> network = instance.ocl_network;
+
+    // Process input parameters. Correctness of parameters was validated by validate_call.
+    // Since we have no correlation between Function::m_parameters and inputs, there is
+    // we try to match them by index number in vectors.
+    for (size_t i = 0; i < inputs.size(); i++)
+    {
+        shared_ptr<runtime::intelgpu::IntelGPUTensorView> tv =
+            static_pointer_cast<runtime::intelgpu::IntelGPUTensorView>(inputs[i]);
+        const op::ParameterVector& input_params = func->get_parameters();
+        network->set_input_data(input_params[i]->get_output_tensor().get_name(),
```

Comment:
You could replace `get_name()` with something that is not a string. Can you just use its position in the input vector? I just don't like strings for performance reasons but it is probably not an issue.

Reply:
Does this copy the data?

Reply:
We should use std::string here because we have to provide the name to cldnn::set_input_data. clDNN uses std::string for  primitive_id also.

We aren't sure but it doesn't matter because method network.execute() is followed the input data.